index.html

```diff
@@ -1777,6 +2109,131 @@ <h2>
+        </dd>
+        <dt>
+          "<dfn>pickup</dfn>"
+        </dt>
+        <dd>
+          This refers to the [=physical address|address=] being collected as part of a service
+          pickup. For example, this could be the address for laundry pickup.
+        </dd>
+      </dl>
+    </section>
+    <section data-dfn-for="PaymentOptions">
+      <h2>
+        <dfn>PaymentOptions</dfn> dictionary
+      </h2>
+      <pre class="idl">
+        dictionary PaymentOptions {
+          boolean requestPayerName = false;
+          boolean requestBillingAddress = false;
+          boolean requestPayerEmail = false;
+          boolean requestPayerPhone = false;
+          boolean requestShipping = false;
+          PaymentShippingType shippingType = "shipping";
+        };
+      </pre>
+      <p class="note">
+        The {{PaymentOptions}} dictionary is passed to the {{PaymentRequest}}
+        constructor and provides information about the options desired for the
+        payment request.
+      </p>
+      <dl>
+        <dt>
+          <dfn>requestBillingAddress</dfn> member
+        </dt>
+        <dd>
+          A boolean that indicates whether the <a>user agent</a> SHOULD collect
+          and return the billing address associated with a <a>payment
+          method</a> (e.g., the billing address associated with a credit card).
+          Typically, the user agent will return the billing address as part of
+          the {{PaymentMethodChangeEvent}}'s
+          {{PaymentMethodChangeEvent/methodDetails}}. A merchant can use this
+          information to, for example, calculate tax in certain jurisdictions
+          and update the displayed total. See below for privacy considerations
+          regarding <a href="#user-info">exposing user information</a>.
+        </dd>
+        <dt>
+          <dfn>requestPayerName</dfn> member
+        </dt>
+        <dd>
+          A boolean that indicates whether the <a>user agent</a> SHOULD collect
+          and return the payer's name as part of the payment request. For
+          example, this would be set to true to allow a merchant to make a
+          booking in the payer's name.
+        </dd>
+        <dt>
+          <dfn>requestPayerEmail</dfn> member
+        </dt>
+        <dd>
+          A boolean that indicates whether the <a>user agent</a> SHOULD collect
+          and return the payer's email address as part of the payment request.
+          For example, this would be set to true to allow a merchant to email a
+          receipt.
+        </dd>
+        <dt>
+          <dfn>requestPayerPhone</dfn> member
+        </dt>
+        <dd>
+          A boolean that indicates whether the <a>user agent</a> SHOULD collect
+          and return the payer's phone number as part of the payment request.
+          For example, this would be set to true to allow a merchant to phone a
+          customer with a billing enquiry.
+        </dd>
+        <dt>
```

Comment:
Because the values of payer name, telephone, and email are simple, we are not going to refer to the Contact Picker API enum.

Reply:
✅